### PR TITLE
agrougrou command

### PR DESCRIPTION
**Pull request by lucasriondel**

### CODE REVIEW

File: src/commands/agrougrou.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+const help: ICommand = {
+    name: "agrougrou",
+    aliases: ["ahouuu", "wofe"],
+    description: "AGROUGROU",
+    usage: "",
+
+    cooldown: 1,
+    isGuildOnly: false,
+    hasArgs: true,
```
Comment (SimonGalet) [inline src/commands/agrougrou.ts]: This command doesn't have args

File: src/commands/agrougrou.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    minimumArgsNb: 0,
+    hasParameters: false,
+
+    execute(message: Message, args: string[]): void {
```
Comment (SimonGalet) [inline src/commands/agrougrou.ts]: This command doesn't have args

File: src/commands/agrougrou.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        const commands = SCommands.Instance.commands;
```
Comment (SimonGalet) [inline src/commands/agrougrou.ts]: Why ?

**Review by SimonGalet — COMMENTED** [review]
jtm

File: src/commands/agrougrou.ts (hunk 3 above)
Comment (lucasriondel) [reply src/commands/agrougrou.ts]: tg

File: src/commands/agrougrou.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import {Message} from "discord.js";
+import ICommand from "../interfaces/ICommand";
+import SCommands from "../singletons/SCommands";
```
Comment (SimonGalet) [inline src/commands/agrougrou.ts]: Useless

File: src/commands/agrougrou.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+import SConfig from "../singletons/SConfig";
```
Comment (SimonGalet) [inline src/commands/agrougrou.ts]: Same

File: src/commands/agrougrou.ts [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+const prefix = SConfig.Instance.prefix;
```
Comment (SimonGalet) [inline src/commands/agrougrou.ts]: Same

**Review by SimonGalet — APPROVED** [review]

**Pull request merged**